containers/Investment/Tranche/index.tsx

```diff
@@ -66,7 +66,7 @@ class TrancheView extends React.Component<Props> {
           {isInvestor &&
             <Box margin={{ top: 'medium', bottom: 'large' }} >
               <Box gap="medium" align="start" margin={{ bottom: 'medium' }} >
-                <Heading level="4" margin="none">Invest / Redeem </Heading>
```

Comment:
We agreed to use Invest as a user facing term. In the contracts we are using supply though

Reply:
Reverted. I used the terms that @denniswell had in his screenshots, but it seems they were outdated.